deploy/olm-catalog/keda/0.0.4/keda.v0.0.4.clusterserviceversion.yaml

```
@@ -0,0 +1,248 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: keda.v0.0.4
+  namespace: placeholder
+  annotations:
+    alm-examples: >-
+      [{"apiVersion":"keda.k8s.io/v1alpha1","kind":"ScaledObject","metadata":{"name":"SCALED-OBJECT-NAME","labels":{"deploymentName":"DEPLOYMENT-NAME"}},"spec":{"scaleTargetRef":{"deploymentName":"DEPLOYMENT-NAME","containerName":"example-container"},"pollingInterval":30,"cooldownPeriod":300,"minReplicaCount":0,"maxReplicaCount":100,"triggers":null}},{"apiVersion":"keda.k8s.io/v1alpha1","kind":"TriggerAuthentication","metadata":{"name":"keda-trigger-auth-azure-queue-secret","namespace":"keda"},"spec":{"podIdentity":{"provider":"none
+      | azure | gcp |
+      spiffe"},"secretTargetRef":[{"parameter":"connectionString","name":"my-keda-secret-entity","key":"azure-storage-connectionstring"}],"env":[{"parameter":"region","name":"my-env-var","containerName":"my-container"}]}},{"apiVersion":"keda.k8s.io/v1alpha1","kind":"TriggerAuthentication","metadata":{"name":"example-triggerauthentication"},"spec":null},{"apiVersion":"keda.k8s.io/v1alpha1","kind":"ScaledObject","metadata":{"name":"example-scaledobject"},"spec":null}]
+    categories: Cloud Provider
+    certified: 'false'
+    description: Kubernetes-based Event Driven Autoscaling
+    containerImage: ''
+    support: ''
+    capabilities: Basic Install
+    repository: 'https://github.com/kedacore/keda'
+    createdAt: 2019-10-30T00:00:00.000Z
+spec:
+  displayName: KEDA
+  description: >+
+    KEDA is a Kubernetes-based event driven autoscaler.  KEDA can monitor event
+    sources like Kafka, RabbitMQ, or cloud event sources and feed the metrics
+    from those sources into the Kubernetes horizontal pod autoscaler.  With
+    KEDA, you can have event driven and serverless scale of deployments within
+    any Kubernetes cluster.
+
+
+  maturity: alpha
+  version: 0.0.4
+  replaces: ''
+  minKubeVersion: '1.11.0'
+  keywords:
+    - keda
+    - scaling
+    - event
+  maintainers:
+    - name: Keda Community
+      email: keda-core@googlegroups.com
+  provider:
+    name: Microsoft and Red Hat
```

Maybe just list KEDA?

```suggestion
    name: KEDA
```